installers/osie/mirror.go

```diff
@@ -14,11 +14,13 @@ const (
 )
 
 var (
-	osieURL                            = mustBuildOSIEURL().String()
-	mirrorBaseURL                      = conf.MirrorBaseUrl
-	dockerRegistry                     string
-	grpcAuthority, grpcCertURL         string
-	registryUsername, registryPassword string
+	osieURL                                 = mustBuildOSIEURL().String()
+	mirrorBaseURL                           = conf.MirrorBaseUrl
+	dockerRegistry                          string
+	grpcAuthority, grpcCertURL              string
+	registryUsername, registryPassword      string
+	log_driver, log_tag, log_server_address string
+	log_server_address_type                 string
```

Comment:
not sure why these are multiple vars in one line, especially log_server_address and log_server_address_type in different lines.

Reply:
I followed the existing implementation as in line no. 21.

Reply:
Can you actually split this off into one item per line. This form of variable declaration isn't really done any where else and should not have been done this way here.

Reply:
done